### PR TITLE
[UTOPIA-701]update ppq form to use dropdown common component 

### PR DESCRIPTION
**Pull request by binggaofw**
# Description

This PR includes the following proposed change(s):

- ppq form does not use our common dropdown component so the icon can not click, I switch to our common dropdown component now

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Development Dependency Working Agreement
- [x] My code DOES NOT include the importing of new dependencies into the DPIA ecosystem
- [ ] My code DOES include the importing of new dependencies into the DPIA ecosystem
**If new dependencies are being introduced to the DPIA ecosystem:**
- [ ] The functionality of the dependency drastically reduces code complexity and makes my changes more easily maintainable and readible 
- [ ] The dependency being introduced does not contain multiple layers of nested dependencies introducing maintainability complexity to the DPIA ecosystem

## Frontend Development Changes
- [ ] N/A
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to project documentation or diagrams that reflect my changes
- [ ] New and existing unit tests pass locally with my changes
- [ ] My code follows Airbnb React Style Guidelines

## API Development Changes
- [x] N/A
- [ ] I have performed a self-review of my own code
- [ ] My code follows standards and practices outlined in the BC Government API Development Guidelines
- [ ] New and existing unit tests pass locally with my changes
- [ ] My changes includes Swagger documentation updates that reflect the changes I am introducing

## Definition of Done

Definition of Done

### CODE REVIEW

File: src/frontend/src/pages/PPQFormPage/PPQFormPage.tsx [hunk 1]
```diff
@@ -134,32 +135,16 @@ const PPQFormPage = () => {
               />
             </div>
             <div className="row">
```
Comment (kushal-arora-fw) [inline src/frontend/src/pages/PPQFormPage/PPQFormPage.tsx:137]: You may not need this div as well. Row without col is useless and uses extra margins.

Comment (kushal-arora-fw) [reply src/frontend/src/pages/PPQFormPage/PPQFormPage.tsx:137]: Ignore this. You're using it as an optional class. My bad.

However, if you look at the classes in the developer console, there are two `form-group`. One from your line #138 and other from Dropdown common component.

You can remove line #138 instead.

Comment (binggaofw) [reply src/frontend/src/pages/PPQFormPage/PPQFormPage.tsx:137]: done

**Review by kushal-arora-fw — APPROVED** [review]

**Comment by kushal-arora-fw** [comment]
Sorry, my merge introduced Conflicts :/

**Comment by binggaofw** [comment]
@kushal-arora-fw it is ok, could you please approve the pr

**Comment by kushal-arora-fw** [comment]
Weirdly, even after merging main here - I do not see my migration in the code :/

**Comment by kushal-arora-fw** [comment]
ohh, no my bad! Too many branches. Looks good. Approved.

**Review by kushal-arora-fw — APPROVED** [review]

**Pull request merged**